src/cli/cli.cpp

```diff
@@ -266,7 +266,15 @@ void parsePgnOut(const std::vector<std::string> &params, ArgumentData &argument_
         });
     } catch (const std::exception &e) {
         // try to read as cutechess pgnout
-        parseValue(params, argument_data.tournament_options.pgn.file);
+        std::string val = concat(params);
+        if (str_utils::contains(val, " ")) {
+            const auto string_vector = str_utils::splitString(val, ' ');
```

Comment:
you concat the params to just split the string again?

Reply:
ahh yeah that was from before the refactor, fixed it